iommi/templates/iommi/table/row_group.html

```diff
@@ -0,0 +1,5 @@
+<tr>
+    {{ traversable.iommi_open_tag }}
```

Comment:
Should the `row_group` name be used here as well, since this is the one users will copy and modify.